memtx: fix a bug with mvcc and exclude_null option (#9955)
tarantool/tarantool: changelogs/unreleased/gh-9954-mvcc-crash-with-exclude-null.md

## bugfix/core

> * Fixed a bug that would lead to crash when both MVCC and index with `exclude_null` part were used (gh-9954).

Reviewer: ```suggestion
* Fixed a bug that resulted in a crash when both MVCC and index with the `exclude_null` part were used (gh-9954).
```
